executors/cmdexec/fakeexec_test.go

```diff
@@ -51,6 +51,7 @@ func NewFakeExecutor(f *CommandFaker) (*FakeExecutor, error) {
 	t.fake = f
 	t.Fstab = "/my/fstab"
 	t.portStr = "22"
+	t.MountOpts = "rw,inode64,noatime,nouuid"
```

Comment:
Reuse the new default global  you defined unless you mean to test this with a custom string for some reason.